modules/apiserversource-kn.adoc

```diff
@@ -113,10 +136,34 @@ Data,
 [id="delete-apiserversource-kn_{context}"]
 == Deleting the ApiServerSource
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `kn` and `oc` commands:
+. Delete the trigger by entering the following command:
++
+
+[source,terminal]
+----
+$ kn trigger delete <trigger-name>
+----
+
+. Delete the service by entering the following command:
++
+
+[source,terminal]
+----
+$ kn service delete <service-name>
+----
+
+. Delete the event source by entering the following command:
++
+
+[source,terminal]
+----
+$ kn source apiserver delete <source-name>
+----
+
+. Delete the service account, cluster role, and cluster binding created in this guide by entering the following command:
```

Comment:
Agree with this and suggest removing "this guide" occurrences throughout and replacing with "this section" if necessary.